### DISCUSSION TITLE
Nemo configurations via the terminal

### DISCUSSION THREAD

**Discussion by johncarterofmars** [linuxmint/discussions#224] [Questions]
I'm looking for a way to make config changes to Nemo via the terminal so that I can add them to a bash script. For example, to set the view to show the number of items in a folder. Rather than going through Preferences, can this be done via a terminal?

Thank you.

**Comment by mtwebster** [comment]
You can view these settings using the dconf-editor (once installed, navigate to org.nemo). And all of them can be modified using the gsettings tool (should already be installed).

An alternative to this (especially if you're pre-configuring multiple machines, you can use an override file.
```
ls -la /usr/share/glib-2.0/schemas/*.override
```
In one we do for mint, we already do some overrides:
```
[org.nemo.desktop]
computer-icon-visible=false
home-icon-visible=false
network-icon-visible=false
trash-icon-visible=false
font="Ubuntu 10"
volumes-visible=true

[org.nemo.window-state]
start-with-status-bar=true

[org.nemo.preferences]
thumbnail-limit=34359738368
bulk-rename-tool=b'bulky'
```

**Comment by johncarterofmars** [comment]
This is very helpful. I didn't know they could be accessed in gsettings. I should have looked. Sorry about that. But this is exactly what I needed. 
Thank you very much.
